README.md

```diff
@@ -16,7 +16,14 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+When an element is semantic it gives structure to webpage so that the browser and the developer understands it purpose.
```

Comment:
`<p`> actually is a semantic element. Semantic elements tell you what the text *does*: what its purpose is.